profile/README.md

```diff
@@ -0,0 +1,17 @@
+# freeCodeCamp
+
+<img src="https://cdn.freecodecamp.org/platform/universal/fcc_meta_1920X1080-indigo.png" align="center" alt="freeCodeCamp's logo" />

```

Comment:
This is a pretty gargantuan image, I would recommend something smaller in height. I am of the opinion that the twitter header we have (same image on the main repo) is the most appealing - even though the logo is old.

Reply:
Replaced!